docs/user-guide/src/SUMMARY.md

```diff
@@ -14,6 +14,7 @@
    - [Basic Features](bmo/features.md)
       - [Automated Cleaning](bmo/automated_cleaning.md)
       - [Automatic Secure Boot](bmo/automatic_secure_boot.md)
+      - [Firmware Components](bmo/firmware_components.md)
```

Comment:
I'd prefer to call this "Firmware Updates" - this is what our users will be looking for.